taar_lite/recommenders/guid_based_recommender.py

```diff
@@ -47,9 +52,13 @@ def can_recommend(self, client_data):
 
     def recommend(self, client_data, limit):
```

Comment:
Can we please specify a default value of 4 if limit is not passed. Also, comment that justification as being driven by the AMO UX team's choice of how these will be surfaced. Currently 4 recs covers all of TAAR-lite's use cases.

Reply:
Added a default of 4 for this.